app/services/testdata/Phase2TestsExpiredFromStartedStatusGenerator.scala

```diff
@@ -16,11 +16,12 @@
 
```

Comment:
As discusses, I think what we have right now, supports the application status but not the progress status.